server/@types/approved-premises/index.d.ts

```diff
@@ -245,5 +246,14 @@ declare module 'approved-premises' {
       daysSinceApplicationRecieved: number
       status: 'In progress' | 'Submitted' | 'Information Requested' | 'Rejected'
     }
+    Application: {
+      id: string
+      crn: string
+      createdByProbationOfficerId: string
+      schemaVersion: string
+      createdAt: string
+      submittedAt?: string
+      data: string
+    }
```

Comment:
We also have an `Application` type now in the form-pages. It might be worth changing that to `ApplicationData` and giving the `data` a type of `ApplicationData`